code/modules/research/protolathe.dm

```diff
@@ -106,6 +105,8 @@ Note: Must be placed west/left of and R&D console to function.
 		if(!in_range(src, stack) || !user.Adjacent(src))
 			return
 		var/amount_inserted = materials.insert_stack(O,amount)
+		if (amount_inserted <= 0)

```

Comment:
I actually have no idea and this just looked like a thing that would work. I tested though and it seems to have fixed it.